app/views/documents/edit.html.erb

```diff
@@ -0,0 +1,44 @@
+<div class="container slice-xs">
+  <div class="container">
+    <div class="row justify-content-center">
+      <div class="card form-card form-card--style-1">
+        <div class="form-body">
+          <h2 class='text-center px-2 heading heading-2 strong-400 mb-0'>
+            <%= t '.document_edit' %>
+          </h2>
+          <%= form_with model: @document, local: true do |form| %>
+            <div class="col">
+              <div class="col-lg-12">
+              </br>
+                <div class="field">
+                  <%= form.label :category, t('activerecord.attributes.document.category')%><br />
+                  <%= form.select :category, Document.category.options, :selected => @document.category, class: "form-control form-control-lg" %>
+                </div>
+                <br/>
+
+                <div class="d-flex justify-content-between flex-wrap">
+                  <div class="field flex-fill">
+                    <%= form.label :file_name, t('activerecord.attributes.document.filename') %>
+                    <br />
+                    <%= form.text_field :file_name, class: "form-control form-control-lg" %>
+                  </div>
+
+                  <div class="mx-2"></div>
+
+                </div>
+                <br>
+
+              </div>
+            </div>
+            <br />
+
+
+            <div class="actions">
+              <%= form.submit t("main_app.views.application.buttons.edit_submit"), class: "btn-update btn btn-styled btn-lg btn-block btn-base-1 mt-4" %>
```

Comment:
How would `t('.edit_submit')` work here ?

Reply:
regarding this comment, is it then best practice to add another key to views/documents so it's clear it's scoped to this view? .edit_submit does not work here.

Reply:
The `missing translation` error should actually guide you on wherei to put the required locale key.

e.g.
`translation missing : 'documents.edit.edit.submit` would mean you should put the locale as

```yml
fr:
  documents:
    edit:
      edit_submit: 'your string here'
```

Reply:
btw, since the locale scope would already be under an `edit` key, you could simplify the key name : 
```yml
fr:
  documents:
    edit:
      submit: 'your string'
```